Library/Formula/chromaprint.rb

```diff
@@ -21,4 +21,9 @@ def install
     system "cmake", ".", *args
     system "make", "install"
   end
+
+  test do
+    # will not work if built without examples
```

Comment:
@mikemcquaid I meant to test if that works in test blocks but then it slipped my mind. Good to know, thanks.